src/Core/Commander/Providers/IoDriver_Image.js

```diff
@@ -18,32 +18,19 @@ define('Core/Commander/Providers/IoDriver_Image', ['Core/Commander/Providers/IoD
     IoDriver_Image.prototype.constructor = IoDriver_Image;
 
     IoDriver_Image.prototype.read = function(url) {
-
         return new Promise(function(resolve/*, reject*/) {
-
             var image = new Image();
 
             image.addEventListener('load', function(/*event*/) {
-
                 resolve(this);
-
-            }, false);
-
-            image.addEventListener('progress', function(/*event*/) {
-
             }, false);
 
-
             image.addEventListener('error', function(/*event*/) {
-
-                this.src = '';
                 resolve(undefined);
-
-            }.bind(this), false);
+            }, false);
```

Comment:
Unless the small changes (remove `progress` event handling, remove setting `src` to the empty-string) are needed (they might very-well be), maybe revert changes to this file?

(either that or put them in a specific commit: this is no longer about standardizing on XHR or promise, this is already using promise; it's possibly –probably?– broken, but that's a different issue, so it deserves a distinct commit IMO)